docs/guides/workspace-and-project-management.rst

```diff
@@ -0,0 +1,39 @@
+================================
+Workspace and Project Management
+================================
```

Comment:
**nitpick**: I find the word "management" misleading, because I'm used to the word "managing" denoting a full CRUD. From the SDK perspective we only (R)ead the workspace and project. Any manipulation, like creating new or removing or editing them requires booting up Portal or asking a Platform admin for help.

How about this?

```suggestion
=======================
Workspaces and Projects
=======================
```